### ISSUE TITLE
Opening issues that asking for collaboration on specific tasks

### ISSUE THREAD

**Issue by luzpaz** [HarryvL/fcFEM-basic#7]
It would be great to add ticket numbers next to each [TODO](https://github.com/HarryvL/fcFEM-basic#todo) that way folks who have skills or want to help in some way can follow the discussions and progress of each endeavor

**Comment by HarryvL** [comment]
Good idea, but I am not quite ready to manage that level of in-depth contribution yet. The TODO's for now are "notes to self". When I am far enough down the original README-TODO list I will try to set that up.
